contractcourt/briefcase.go

```diff
@@ -62,8 +62,10 @@ type ArbitratorLog interface {
 
 	// InsertUnresolvedContracts inserts a set of unresolved contracts into
 	// the log. The log will then persistently store each contract until
-	// they've been swapped out, or resolved.
-	InsertUnresolvedContracts(...ContractResolver) error
+	// they've been swapped out, or resolved. It takes a report which should
+	// be written to disk if as well if it is non-nil.
+	InsertUnresolvedContracts(report *channeldb.ResolverReport,
```

Comment:
This seems mismatched: we're inserting a _series_ of contracts here (which will each eventually generate their _own_ report), but then accept a _single_ resolver report?